operations/network/management_network/apply_custom_config_1.2.md

```diff
@@ -1,20 +1,21 @@
-# Apply Custom Switch Config CSM 1.2
+# Apply Custom Switch Configuration CSM 1.2
 
 Apply the backed up site connection configuration with a couple modifications. Since virtual routing and forwarding (VRF) is now used to separate customer traffic, the site ports and default routes must be added to that VRF.
 
 ## Prerequisites
 
 - Access to the switches
 - Custom switch configurations
-    - [Backup Custom Config](backup_custom_config.md)
+  - [Backup Custom Configuration](backup_custom_configurations.md)
+
 - Generated switch configurations already applied
-    - [Apply Switch Configs](apply_switch_configs.md)
+  - [Apply Switch Configurations](apply_switch_configurations.md)
 
 ## Aruba
```

Comment:
The later Aruba/Mellanox/etc sections have headings which explain what is being done in those sections. The same should be done here.